resources/maskinporten.py

```diff
@@ -218,6 +220,15 @@ def delete_client(
             f"Client {client_id} cannot be deleted due to internal server error.",
         )
 
+    send_to_aws = body.aws_account and body.aws_region
```

Comment:
Er ikke `body` (`DeleteMaskinportenClientIn`) satt som optional over?

Reply:
True! Fikser og legger til en test for det.